### ISSUE TITLE
Getting error when we pass double quoted search terms with "extract_metadata" = True.

### ISSUE THREAD

**Issue by thamizhchelvan** [hardikvasa/google-images-download#170]
Hello,

Thanks for the nice work. I am getting error, when I pass double quoted search terms with "extract_metadata" = True.

Payload: 
`{"keywords":"\"siberian husky\"","limit":1,"print_urls":True,"extract_metadata":True,"usage_rights":"labeled-for-reuse","no_directory":True, "no_numbering":True}`

Error:
```

Traceback (most recent call last):
  File "googleImage.py", line 6, in <module>
    gresponse = response.download(arguments)
  File "AppData\Local\Programs\Python\Python37-32\lib\site-packag
es\google_images_download\google_images_download.py", line 858, in download
    json_file = open("logs/"+search_keyword[i]+".json", "w")
OSError: [Errno 22] Invalid argument: 'logs/"siberian husky".json'
```

**Comment by hardikvasa** [comment]
@thamizhchelvan sorry for the delayed response. Were you able to get around this issue?
what OS version and flavor you are testing this on? Also why not just use keywords without extra quotes like this `"keywords":"siberian husky"` ?

i however tested the CLI equivalent of what you re trying
`python3 google_images_download.py -k "\"siberian husky\"" -l 1 --print_urls --extract_metadata --usage_rights 'labeled-for-reuse' --no_directory --no_numbering`

but i did not get any error.   Maybe the os doesn't like the .json file name to have double quotes?

**Comment by thamizhchelvan** [comment]
@hardikvasa ,

Nope, still having the same issue. I am using double quotes to match the exact terms inside the quotes. I am using Windows 7.

**Comment by WR-BillRich** [comment]
How about editing the code so that the directory name (dir_name) only contains code that is compatible with windows directory naming?
![image](https://user-images.githubusercontent.com/49466088/59160487-f9e45200-8b08-11e9-994d-377721a0515e.png)
By using the above function, we change the strip the illegal symbol from dir_name in both create_directories def and get_all_item def
![image](https://user-images.githubusercontent.com/49466088/59160509-44fe6500-8b09-11e9-8c7d-dc75e40834c2.png)
![image](https://user-images.githubusercontent.com/49466088/59160514-5c3d5280-8b09-11e9-96df-5622e45dd989.png)
